### ISSUE TITLE
Custom Typeface method?

### ISSUE THREAD

**Issue by Darshan10298** [javiersantos/MaterialStyledDialogs#56]
Is there a possibility of Adding a method to use custom typeface on all the text in the Dialog?
Somethin like .setTypeface?
it will be great if included 😀

**Comment by KgotsoK** [comment]
Yeah I also need to customize the font on these dialogs. @Darshan10298 do you have a workaround, considering no such setTypeface method exists?
